.github/workflows/test_iree.yml

```diff
@@ -157,15 +157,15 @@ jobs:
             iree_tests/pytorch/models \
             iree_tests/sharktank \
             -n 4 \
-            -rpfE \
+            -rA \
```

Comment:
This includes logs for XFAIL'd tests.
* Logs: https://github.com/nod-ai/SHARK-TestSuite/actions/runs/9769659230/job/26969524910?pr=277#step:7:24
* Compared to https://github.com/nod-ai/SHARK-TestSuite/actions/runs/9749934013/job/26908189904#step:7:24

I think that's useful to show _why_ the model tests are failing, but the "reason" is confusing in those cases:
> XFAIL iree_tests/pytorch/models/opt-125M/opt-125M.mlirbc::cpu_llvm_task_real_weights - Expected compilation to fail (remove from 'expected_compile_failures')

Normally we only show that "reason" when the XFAIL'd test starts to pass unexpectedly. Hooray double negatives. Could just adjust the language somehow 🤔